<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Janelinha</class>
 <widget class="QDialog" name="Janelinha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lista IGNORA</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo-epe.png</normaloff>logo-epe.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>221</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Barras a serem ignoradas</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_nbarra">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>40</y>
      <width>43</width>
      <height>20</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>Campo para inserção de barras na lista ignora. Apenas números inteiros positivos.</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_incluir">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Incluir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_removerselec">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remover Seleção</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>41</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nº da barra:</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>311</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>Lista de barras que serão ignoradas na criação do arquivo Monitora.dat e da lista de contingências.</string>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
